cmd/root.go

```diff
@@ -45,6 +45,7 @@ var (
 	cfgFile                  string
 	databaseConfig           config.Database
 	diffBlockFromHeadOfChain int64
+	endingBlockNumber        int64
```

Comment:
You probably want this in the backfillEvents file, since it only applies there. It works here, because everything is the same `cmd` package. 

That actually might be a problem since check the two backfills have a lot in common and checkMigrations is this outlier that only cares about the logfile stuff. I don't know if it's worth it to have a further subdirectory for cmd/backfill/<events/urns>.

Reply:
Can definitely move it over 👍